rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -227,6 +229,16 @@ cat > /var/www/rutorrent/conf/config.php <<EOL
 // Set to true to enable rapid cached loading of ruTorrent plugins
 \$cachedPluginLoading = false;
```

Comment:
It's recommended to add an environment variable to change this value. It's stable enough now to change with a 3 hour JavaScript cache duration. I would just add a note, to clear the web browser cache when upgrading versions.

Default `false`, so the user doesn't unknowingly upgrade versions without clearing the web browser cache.

Reply:
Nevermind I see it's already done: https://github.com/Novik/ruTorrent/commit/cba1bfc11cc8ebc8a3c65d1d37cfbd4d6261a39e

Reply:
><script type="text/javascript" src="myfile.js?v4.0.4"></script>
> For the cache issue, could we just not add a query string with the revision so it enforces cache reload for the client?

This cache is related to a combined script where PHP initializes plugin configuration in JavaScript.
https://github.com/Novik/ruTorrent/blob/master/php/getplugins.php

The cache code is located here. https://github.com/Novik/ruTorrent/blob/master/php/utility/cachedecho.php#L11-L24

If you have a solution please free to submit a pull request. So far, I've only been able to mitigate the problem. The difference in loading time is sometimes seconds longer versus 300ms to 500ms with plugin caching enabled.

Reply:
I see some etag handling based on content hash, that sounds about right.